packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/Examples.3/ColumnActions.7.doc.mdx

```diff
@@ -0,0 +1,16 @@
+import ColumnActions from './ColumnActions?dev-site-example';
+
+# Column Actions
+
+### Description
+
+This example demonstrates a [Worklist Data Grid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) that supports Column Actions feature.
+
+#### Required Properties
+
+The **[action](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#column) property** needs to be passed to a column with following properties:
```

Comment:
not sure on the best wording, but can we clarify that `label` and `onClick` are required properties FOR the action property somehow?
```suggestion
The **[action](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#column) property** requires the following properties:
```